README.md

```diff
@@ -39,31 +39,49 @@ function createElem (tagName) {
 var elem = createElem('video')
 
 var readable = // ... get a readable stream from somewhere
-var writable = new MediaSourceStream(elem, { extname: '.webm' })
+var wrapper = new MediaElementWrapper(elem)
+// The correct mime type, including codecs, must be provided
+var writable = wrapper.getStream('video/webm; codecs="vorbis, vp8"')
 
 elem.addEventListener('error', function (err) {
   // listen for errors on the video/audio element directly
 })
 
+writable.on('error', function (err) {
+  // more detailed error messages may also be provided on the stream,
+  // but you won't miss any errors if you only listen on the element
+  // directly
```

Comment:
On second thought, the last suggest to listen only for errors on the stream doesn't make sense since the stream doesn't re-emit errors from the element.